### PR TITLE
Chapter 7: Sign up

### PR DESCRIPTION
**Pull request by NguyenDuc95HT**
image

### CODE REVIEW

**Comment by NguyenDuc95HT** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,3 +1,25 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
```
Comment (phamvanbk11) [inline app/controllers/users_controller.rb]: Không tìm thấy user thì sao?

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params) # Not the final implementation!
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to @user
+    else
+      render "new"
```
Comment (phamvanbk11) [inline app/controllers/users_controller.rb]: :new

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params) # Not the final implementation!
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
```
Comment (phamvanbk11) [inline app/controllers/users_controller.rb]: I18n

**Comment by phamvanbk11** [comment]
commented

**Comment by NguyenDuc95HT** [comment]
ready

**Comment by dothidiemthao** [comment]
merged

**Pull request merged**